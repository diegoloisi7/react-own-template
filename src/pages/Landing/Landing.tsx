import reactLogo from "@assets/react.svg"
import viteLogo from "/vite.svg"
import "./Landing.css"
import { Organism } from "@components/index"
import { FC } from "react"

export const Landing: FC = () => {
	return (
		<>
			<div>
				<a href="https://vite.dev" target="_blank" rel="noreferrer">
					<img src={viteLogo} className="logo" alt="Vite logo" />
				</a>
				<a href="https://react.dev" target="_blank" rel="noreferrer">
					<img src={reactLogo} className="logo react" alt="React logo" />
				</a>
			</div>
			<h1>Vite + React</h1>
			<Organism text="Template para ahorrar tiempo ğŸ˜›" />
		</>
	)
}
