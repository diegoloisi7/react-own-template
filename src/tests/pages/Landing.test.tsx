import { Landing } from "@pages/Landing/Landing"
import { render } from "@testing-library/react"

describe("Landing test", () => {
	it("should test Landing", () => {
		const component = render(<Landing />)
		expect(component).toMatchSnapshot()
	})
})
