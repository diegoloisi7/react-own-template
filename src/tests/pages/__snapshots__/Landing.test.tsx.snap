// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing test should test Landing 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <a
          href="https://vite.dev"
          rel="noreferrer"
          target="_blank"
        >
          <img
            alt="Vite logo"
            class="logo"
            src="test-file-stub"
          />
        </a>
        <a
          href="https://react.dev"
          rel="noreferrer"
          target="_blank"
        >
          <img
            alt="React logo"
            class="logo react"
            src="test-file-stub"
          />
        </a>
      </div>
      <h1>
        Vite + React
      </h1>
      <div>
        Template para ahorrar tiempo ðŸ˜›
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <a
        href="https://vite.dev"
        rel="noreferrer"
        target="_blank"
      >
        <img
          alt="Vite logo"
          class="logo"
          src="test-file-stub"
        />
      </a>
      <a
        href="https://react.dev"
        rel="noreferrer"
        target="_blank"
      >
        <img
          alt="React logo"
          class="logo react"
          src="test-file-stub"
        />
      </a>
    </div>
    <h1>
      Vite + React
    </h1>
    <div>
      Template para ahorrar tiempo ðŸ˜›
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
